### PR TITLE
Fix simultaneous display of OverlayCamera and OverlayImage

### PR DESCRIPTION
**Pull request by peci1**
Without this change, adding both OverlayCamera and OverlayImage to one rviz window crashes with:

```
[ WARN] [1669695420.835465329, 1666865132.227821766] [ros.rviz]: OGRE EXCEPTION(4:ItemIdentityException): Overlay with name 'OverlayImageDisplayObject0' already exists! in OverlayManager::create at /build/ogre-1.9-CZD0Iy/ogre-1.9-1.9.0+dfsg1/Components/Overlay/src/OgreOverlayManager.cpp (line 109)
terminate called after throwing an instance of 'Ogre::ItemIdentityException'
  what():  OGRE EXCEPTION(4:ItemIdentityException): Overlay with name 'OverlayImageDisplayObject0' already exists! in OverlayManager::create at /build/ogre-1.9-CZD0Iy/ogre-1.9-1.9.0+dfsg1/Components/Overlay/src/OgreOverlayManager.cpp (line 109)
```

I'm not really sure whether it is correct that I made this string different from the similar one used in the OverlayCamera (`OverlayCameraDisplayObject`). If these two should be the same, I can update the PR.

### CODE REVIEW

**Review by knorth55 — APPROVED** [review]

File: jsk_rviz_plugins/src/overlay_camera_display.cpp [hunk 1]
```diff
@@ -398,7 +398,7 @@ void OverlayCameraDisplay::update( float wall_dt, float ros_dt )
   if (!overlay_) {
     static int count = 0;
     rviz::UniformStringStream ss;
-    ss << "OverlayImageDisplayObject" << count++;
+    ss << "OverlayCameraImageDisplayObject" << count++;
```
Comment (knorth55) [inline jsk_rviz_plugins/src/overlay_camera_display.cpp:401]: Thank you! this is typo, I think.

can you change like this?

```suggestion
    ss << "OverlayCameraDisplayObject" << count++;
```

as https://github.com/jsk-ros-pkg/jsk_visualization/blob/7d31711959cfe88eaef8fc8a426accaf6dd02621/jsk_rviz_plugins/src/overlay_camera_display.cpp#L182

**Review by knorth55 — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
this PR fixes the typo in `overlay_camera_display.cpp`.

```bash
$ cd jsk_rviz_plugins/src
$ grep -Ei '\"[a-Z]+Object' * -r
linear_gauge_display.cpp:    ss << "LinearGaugeDisplayObject" << count++;
overlay_camera_display.cpp:    ss << "OverlayCameraDisplayObject" << count++;
overlay_camera_display.cpp:    ss << "OverlayImageDisplayObject" << count++;
overlay_diagnostic_display.cpp:      ss << "OverlayDiagnosticDisplayObject" << count++;
overlay_image_display.cpp:        ss << "OverlayImageDisplayObject" << count++;
overlay_menu_display.cpp:      ss << "OverlayMenuDisplayObject" << count++;
overlay_text_display.cpp:      ss << "OverlayTextDisplayObject" << count++;
pie_chart_display.cpp:    ss << "PieChartDisplayObject" << count++;
string_display.cpp:      ss << "StringDisplayObject" << count++;

```

File: jsk_rviz_plugins/src/overlay_camera_display.cpp (hunk 1 above)
Comment (peci1) [reply jsk_rviz_plugins/src/overlay_camera_display.cpp]: Changed and tested.

**Review by knorth55 — APPROVED** [review]

File: jsk_rviz_plugins/src/overlay_camera_display.cpp (hunk 1 above)
Comment (peci1) [reply jsk_rviz_plugins/src/overlay_camera_display.cpp]: Unfortunately, with this change, it no longer segfaults, but it also doesn't draw any Rviz items except for the image itself. I'll revert this change.

**Pull request merged**